Makefile

```diff
@@ -234,7 +234,7 @@ unsigned-android: stop pre-build check-style prepare-android-build ## Build an u
 test: | pre-run check-style ## Runs tests
 	@npm test
 
-build-pr: | can-build-pr stop pre-buil check-style ## Build a PR from the mattermost-mobile repo
+build-pr: | can-build-pr stop pre-build check-style ## Build a PR from the mattermost-mobile repo
```

Comment:
this is just to fix the build, will have to submit it in another PR and then rebase this one